eve_sqlalchemy/validation.py

```diff
@@ -156,12 +156,11 @@ def _validate_dependencies(self, document, dependencies, field,
         dcopy = None
         if self._original_document:
             dcopy = copy.copy(document)
-            # Use dict_update and remove_none_values from utils, so existing
-            # values in document don't get overridden by the original document
-            # and None values are removed. Otherwise handling in parent method
+            # Use dict_update from utils, so existing
+            # values in document don't get overridden by the original document.
+            # Otherwise handling in parent method
```

Comment:
It's good you didn't forget about updating that comment, but please fix the formatting, too :).